### ISSUE TITLE
How can I lint without having a tslint.json file?

### ISSUE THREAD

**Issue by justinmchase** [panuhorsmalahti/gulp-tslint#61]
When I run `$ tslint app/main.ts` from the commandline I get a couple of errors about using `'` instead of `"`.

When I run via gulp I do not. I created a `tslint.json` file and then reran it and the errors showed up as expected. I would like to run `gulp-tslint` without having a `tslint.json` file however. I just want it to use all of the default rules instead. I've been trying to figure out how to do this but the documentation doesn't make it clear how to run without one.

Additionally, I would recommend, without knowing more, that the default behavior is to apply the default rules if no tslint.json file is found and require no extra configuration than the basic example given at the top of the README.

Details:

gulp-tslint version: 4.3.4
tslint version: 3.6.0
Operating system: osx

Example gulp configuration (if applicable):
```
gulp.task('typescript:lint', () => {
    return gulp.src('app/main.ts')
        .pipe(tslint())
        .pipe(tslint.report('prose'))
})
```

Error console output:
```
10:51:26:justin:~/code/test (master)$ gulp typescript:lint
[10:51:29] Using gulpfile ~/code/test/gulpfile.js
[10:51:29] Starting 'typescript:lint'...
[10:51:29] Finished 'typescript:lint' after 33 ms
```

TypeScript example code (if applicable):
```
import {bootstrap} from 'angular2/platform/browser';
import {AppComponent} from './app.component';

bootstrap(AppComponent);
```

**Comment by panuhorsmalahti** [comment]
It's a reasonable feature suggestion. Probably should be fixed in #60.

**Comment by valorkin** [comment]
With #63 merged this will be fixed
